neurograph.py

```diff
@@ -0,0 +1,170 @@
+import networkx as nx
```

Comment:
If you're using VS code, install a flake-8 plugin which will enforce some code formatting.

Reply:
I think this file should go under graphing/specialgraphs

Reply:
Resolved both